src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -113,9 +112,6 @@ private void setUpEventHandler() {
 
         // Update keywords
         List<String> all = new ArrayList<>(QualifierType.getCompletionKeywords());
-        all.addAll(e.users.stream()
-            .map(TurboUser::getLoginName)
-            .collect(Collectors.toList()));
```

Comment:
Oh. I kind of liked this feature though. If the concern is the users occupying space in the list when people want to see filter syntax, maybe we could order them last?